src/Chapter1.hs

```diff
@@ -636,7 +649,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n 
+    | div n 10 /= 0 = firstDigit $ div n 10
+    | otherwise = n
```

Comment:
In the case of `-1` it would return `-1` thought it should be `1` 🙂 I would suggest to also look at the negative case 👍🏼